kuttl-test.yaml

```diff
@@ -17,4 +18,5 @@ commands:
 kindConfig: ./kind.yaml
 testDirs:
   - ./tests/e2e/
-timeout: 200
+timeout: 300
```

Comment:
It would be nice if there was some way to override this with and environment variable.

Reply:
IIRC this is not possible right now :(

Reply:
I think the only way to achieve this is using templates and then process the template in the test preparation:

Something like this: https://github.com/mesosphere/kudo-cassandra-operator/blob/master/kuttl-tests/suites/kuttl-common.yaml.template

WDYT? Is an improvement we can do late.

Reply:
I think this can wait until we have these tests running for a while and see what problems we have with timeouts being too long or short.